src/content/docs/data-apis/convert-to-metrics/analyze-monitor-data-trends-metrics.mdx

```diff
@@ -549,12 +549,12 @@ Query operations include:
   </Collapser>
 </CollapserGroup>
 
-## Use the NerdGraph GraphiQL API tool [#use-graphql-api-tool]
+## Use the NerdGraph API explorer tool [#use-graphql-api-tool]
 
-You can use our [GraphiQL tool](https://api.newrelic.com/graphiql) to explore the data structure. You can also use it to build and run the [operations](#operations) to convert events, logs, and spans to metrics. To use this tool:
+You can use NerdGraph API explorer, our GraphiQL tool, to explore the data structure. You can also use it to build and run the [operations](#operations) to convert events, logs, and spans to metrics. To use this tool:
 
 1. Create the metrics operation's request with the required parameters.
-2. Go to **[api.newrelic.com/graphiql](https://api.newrelic.com/graphiql)**, and paste your query into the box.
-3. To execute the operation, press **Play**. Or, to get the cURL format, select **Copy as cURL**.)
+2. Go to **[one.newrelic.com](https://one.newrelic.com) > Apps > NerdGraph API explorer**, and paste your query into the box.
```

Comment:
this is a case when we want to do the /all-capabilities thing for pathing, due to wanting to send them to that page to definitely find the Apps thing.

Reply:
OK, I fixed this.